### DISCUSSION TITLE
Freezed + moor => import error in generated file

### DISCUSSION THREAD

**Discussion by peterhijma** [simolus3/drift#1382] [Q&A]
I have not encountered this error before, so I don't know why I suddenly got this:

```
lib/data/models/time_entry_model.freezed.dart:697:4: Error: 'JsonKey' is imported from both 'package:json_annotation/src/json_key.dart' and 'package:moor/src/dsl/dsl.dart'.
  @JsonKey(ignore: true)
   ^^^^^^^
```

Any help would be appreciated!

EDIT: This is probably moor-independent now that I think of it :(

**Comment by simolus3** [comment]
Do you control the imports in `time_entry_model.freezed.dart` or is that generated as well? You could add an `hide JsonKey` to the import otherwise?

**Reply by peterhijma** [reply]
`time_entry_model.freezed.dart` is the generated file. `time_entry_model.dart` is where I control the imports. But hiding `JsonKey` there doesn't fix the problem it seems?

**Reply by peterhijma** [reply]
You helped me on the right path. I've changed `import 'package:moor/moor.dart';` to `import 'package:moor/moor.dart' show Value;`. That seems to work correctly! :) Thanks for the extremely quick answer once again.
